scipy/interpolate/_polyint.py

```diff
@@ -947,7 +952,7 @@ def barycentric_interpolate(xi, yi, x, axis=0, *, der=0):
     >>> plt.show()
 
     """
-    P = BarycentricInterpolator(xi, yi, axis=axis)
+    P = BarycentricInterpolator(xi, yi, axis=axis, rng=rng)
```

Comment:
Before `rng` was added as a parameter, the PRNG behavior was not documented. Now that we're adding this, enforce the new behavior?
```suggestion
    P = BarycentricInterpolator(xi, yi, axis=axis, rng=np.default_rng(rng))
```

Reply:
Never mind; you're passing `rng` by keyword, so that's what happens.